lib/hearsCheckers/ignoringCode.js

```diff
@@ -0,0 +1,19 @@
+const removeCodeFromMessage = message => message.text.replace(/ *`[^)]*` */g, '')
```

Comment:
@nikitasakov thoughts about using [middleware](https://botkit.ai/docs/v0/middleware.html) here? I think code will be easier to understand with that approach.

Reply:
@nikitasakov I mean we can delete code from message in middleware and process it as usual. It suits a [botkit message pipeline](https://botkit.ai/docs/v0/readme-pipeline.html). What do you think? 🙂

Reply:
@DDKatch Does it mean that we remove code for all incoming messages and can't access it even if we need it in future?

Reply:
@nikitasakov we can access this code via `raw_message` field

Reply:
or even add another field which will keep it

Reply:
@DDKatch Hmm, I like this idea 👍, but as I see this doc for v0 Botkit and when I trying to use it I can't because looks like everything changed in v4 🤔I can't even find something about `message.raw_message` in v4, and almost all middleware seems to work strange (not like in old docs). Have you tried to use middleware? Maybe you can advise something?